mcc/odin/odin/odin_test.go

```diff
@@ -545,6 +545,35 @@ var getRestoreDBInstanceFromDBSnapshotInputCases = []getRestoreDBInstanceFromDBS
 		},
 		expectedError: "",
 	},
+	// Params with Snapshot without OriginalInstanceName
+	{
+		name:       "Params with Snapshot without OriginalInstanceName",
+		identifier: "production",
+		createDBParams: CreateDBParams{
+			DBInstanceType: "db.m1.medium",
+			DBUser:         "owner",
+			DBPassword:     "password",
+			Size:           5,
+		},
+		snapshot: exampleSnapshot1,
+		expectedRestoreDBInstanceFromDBSnapshotInput: nil,
+		expectedError:                                "Original instance is required",
+	},
+	// Params with non existing Snapshot
+	{
+		name:       "Params with non existing Snapshot",
+		identifier: "production",
+		createDBParams: CreateDBParams{
+			DBInstanceType:       "db.m1.medium",
+			DBUser:               "owner",
+			DBPassword:           "password",
+			Size:                 5,
+			OriginalInstanceName: "develop",
+		},
+		snapshot: exampleSnapshot1,
+		expectedRestoreDBInstanceFromDBSnapshotInput: nil,
+		expectedError:                                "Couldn't find snapshot for develop instance",
```

Comment:
the indentation is weird, in this case it's aligned, for the other values above not?